### PR TITLE
CCPA support/compliance

### PR DESCRIPTION
**Pull request by can-soykarafakili**
Both code and GitHub Wiki integration is added. 

For GitHub Wiki please check: https://github.com/pubnative/pubnative-hybid-ios-sdk/wiki/CCPA-Compliance

### CODE REVIEW

**Comment by eros902002** [comment]
We also need to add the support on the client side for CCPA. Like removing all personal info y the user opted out. Check: https://github.com/pubnative/pubnative-hybid-android-sdk/pull/194/files

**Comment by can-soykarafakili** [comment]
@eros902002 Requested feature is added along with COPPA check for `dnt = 1` request param.

File: PubnativeLite/PubnativeLite/Ad Request/PNLiteAdFactory.m [hunk 1]
```diff
@@ -42,7 +43,13 @@ - (PNLiteAdRequestModel *)createAdRequestWithZoneID:(NSString *)zoneID
     adRequestModel.requestParameters[HyBidRequestParameter.coppa] = [HyBidSettings sharedInstance].coppa ? @"1" : @"0";
     [self setIDFA:adRequestModel];
     adRequestModel.requestParameters[HyBidRequestParameter.locale] = [HyBidSettings sharedInstance].locale;
-    if (![HyBidSettings sharedInstance].coppa) {
+    
+    NSString* privacyString =  [[HyBidUserDataManager sharedInstance] getIABUSPrivacyString];
+    if (![[HyBidUserDataManager sharedInstance] isCCPAOptOut] && !([privacyString length] == 0)) {
```
Comment (eros902002) [inline PubnativeLite/PubnativeLite/Ad Request/PNLiteAdFactory.m:48]: We should fix this condition. There's also an error on Android. We should send the consent string if it's set even if the user opts out. So I'd just keep the check to making sure its not nil or empty.

Comment (can-soykarafakili) [reply PubnativeLite/PubnativeLite/Ad Request/PNLiteAdFactory.m:48]: @eros902002 `isCCPAOptOut` is removed in the latest commit. Please check :D

**Pull request merged**